client/index.html

```diff
@@ -13,20 +13,20 @@
 
     <form class="main-section">
 
-        <div>
-            <input id="title" type="text" value="Title">
-            <input id="name" type="text" value="Your name">
-            <input id="story" type="text" value="Your story...">
-        </div>
-        
-        <input id="button" type="submit" value="PUBLISH"> </input>
-    </form>
+        <input type="text" id="title" placeholder="Title"> </br> </br>
+
+        <input type="text" id="writer" placeholder="Name"> </br> </br>
+
+        <input type="text" id="content" placeholder="Tell your story..."> </br> </br>
 
-    <section>
-        
-    </section>
+        <a class="publish">
```

Comment:
Server was running (WELL DONE!) but did not know where new post went? - Maybe DOM Manipulation used for a second page like we did together in our project group Yusra?

Reply:
Thanks! I will work on that next